Testing/Tools/cxxtest/python/cxxtest/cxxtest_parser.py

```diff
@@ -35,7 +35,7 @@ def scanInputFiles(files, _options):
 
 def scanInputFile(fileName):
     '''Scan single input file for test suites'''
-    file = open(fileName)
+    file = open(fileName, encoding='utf-8')
```

Comment:
Confirmed that this seems to solve issues with the CentOS 7 CI build that was defaulting to `ascii`.